Userland/Libraries/LibGfx/QOICommon.h

```diff
@@ -43,4 +45,35 @@ ALWAYS_INLINE static constexpr bool qoi_is_valid_run(u8 run)
     return AK::is_in_bounds(run, QOI_RUN_MIN, QOI_RUN_MAX);
 }
 
+class QOIState {
+public:
+    static constexpr u8 PREVIOUSLY_SEEN_PIXELS_SIZE = 64;
+
+    static inline u8 index_position(Color pixel)
```

Comment:
This is already implicitly `inline` by being in the class definition in the header file.